Sources/Testing/Traits/Trait.swift

```diff
@@ -12,75 +12,80 @@
+  /// - Throws: Any error that prevents the test from running. If an error
+  ///   is thrown from this method, the test is skipped and the error is
+  ///   recorded as an ``Issue``.
   ///
-  /// This method is called after all tests and their traits have been
-  /// discovered by the testing library, but before any test has begun running.
-  /// It may be used to prepare necessary internal state, or to influence
+  /// The testing library calls this method after it discovers all tests and
+  /// their traits, and before it begins to run any tests.
+  /// Use this method to prepare necessary internal state, or to determine
   /// whether the test should run.
   ///
   /// The default implementation of this method does nothing.
   func prepare(for test: Test) async throws
 
-  /// The user-provided comments for this trait, if any.
+  /// The user-provided comments for this trait.
   ///
-  /// By default, the value of this property is an empty array.
+  /// The default value of this property is an empty array.
   var comments: [Comment] { get }
 
   /// The type of the test scope provider for this trait.
   ///
-  /// The default type is `Never`, which cannot be instantiated. The
-  /// ``scopeProvider(for:testCase:)-cjmg`` method for any trait with this
-  /// default type must return `nil`, meaning that trait will not provide a
-  /// custom scope for the tests it's applied to.
+  /// The default type is `Never`, which can't be instantiated. The
+  /// ``scopeProvider(for:testCase:)-cjmg`` method for any trait with
+  /// `Never` as its test scope provider type must return `nil`, meaning that
+  /// the trait doesn't provide a custom scope for tests it's applied to.
   associatedtype TestScopeProvider: TestScoping = Never
 
-  /// Get this trait's scope provider for the specified test and/or test case,
-  /// if any.
+  /// Get this trait's scope provider for the specified test, and optional test
+  /// case.
   ///
   /// - Parameters:
   ///   - test: The test for which a scope provider is being requested.
   ///   - testCase: The test case for which a scope provider is being requested,
   ///     if any. When `test` represents a suite, the value of this argument is
   ///     `nil`.
   ///
-  /// - Returns: A value conforming to ``Trait/TestScopeProvider`` which may be
-  ///   used to provide custom scoping for `test` and/or `testCase`, or `nil` if
-  ///   they should not have any custom scope.
+  /// - Returns: A value conforming to ``Trait/TestScopeProvider`` which you
+  ///   use to provide custom scoping for `test` and/or `testCase`, or `nil` if
+  ///   the trait doesn't provide any custom scope for the test or test case.
   ///
   /// If this trait's type conforms to ``TestScoping``, the default value
   /// returned by this method depends on `test` and/or `testCase`:
   ///
-  /// - If `test` represents a suite, this trait must conform to ``SuiteTrait``.
+  /// - If `test` represents a suite, this trait also conforms to
```

Comment:
The conformance to `SuiteTrait` isn't automatic, hence "must"

Reply:
This is the opposite of @stmontgomery's suggestion:

> What I meant to convey here is that, if test is a suite, that implies this trait conforms to SuiteTrait. It's not something the developer needs to do; it's something which must already be true. Maybe a clearer wording would be
> 
> -- If `test` represents a suite, this trait must conform to ``SuiteTrait`..
> +- If `test` represents a suite, this trait conforms to ``SuiteTrait`.

I can only satisfy one of these, and I think @stmontgomery is correct that if you get here and `test` is a suite, then the trait _already_ conforms to `SuiteTrait` and the developer doesn't need to do anything.

Reply:
This documentation is going to be viewed in isolation, and somebody who reaches it may not realize they're looking at documentation for the _specific_ case where the type already conforms to `SuiteTrait`. Hence we need to make it clear that the described behaviour of this symbol only applies under certain conditions.